tests/Single/ExtraSmallSingleHTMLFormatterTest.php

```diff
@@ -90,10 +90,9 @@ public function testFormatHtmlSingleDateXsMoreDaysCurrentYear(): void
         );
 
         $this->assertEquals(
-            '<span class="cf-from cf-meta">Van</span> ' .
             '<span class="cf-date">23</span> ' .
             '<span class="cf-month">jan</span> ' .
-            '<span class="cf-to cf-meta">tot</span> ' .
+            '- ' .
```

Comment:
Is this not missing a leading space? Should this also be part of a span?

Reply:
The HTML formats that already use a `-` had the `-` inside a span. I guess this can be important if they want to apply their own styling on it.